src/g_editor.c

```diff
@@ -2655,7 +2655,7 @@ int canvas_isconnected (t_canvas *x, t_text *ob1, int n1,
     return (0);
 }
 
-static int canconnect(t_canvas*x, t_object*src, int nout, t_object*sink, int nin)
+int canconnect(t_canvas*x, t_object*src, int nout, t_object*sink, int nin)
```

Comment:
if you make this non-static, you should probably change the name to `canvas_canconnect()` to avoid disambiguities.